File: codewars/go/returning_strings.go
// https://www.codewars.com/kata/55a70521798b14d4750000a4/train/go

package main

import "fmt"

func Greet(name string) string {
    return "Hello, " + name + " how are you doing today?"
}

func main() {
    var name string;
    fmt.Scanf("%s", &name);
    fmt.Println(Greet(name));
}

File: codewars/ocaml/returning_strings.ml
(* https://www.codewars.com/kata/55a70521798b14d4750000a4/train/ocaml *)

let greet(name: string): string =
    "Hello, " ^ name ^ " how are you doing today?"

let main() =
    print_string(greet "bee");;

main()
